hw/ip/spi_device/doc/_index.md

```diff
@@ -309,6 +361,252 @@ All reads and writes to/from the SRAM for RXF and TXF activity are managed by
 direct reads and writes through the TLUL bus interface, managed by the
 auto-generated register file control logic.
 
+## SPI Flash and Passthrough Modes
+
+### Command Information List
+
+The SW may configure the map from the received opcode to the command process module by programming *cmd_info* list.
+Current SPI_DEVICE provides 24 command information entries.
+Each entry represents a command.
+Details of the fields are explained in the {{<regref "CMD_INFO_0">}}
+
+First 11 commands are assigned to specific submodules.
+
+Index  | Assigned Submodule
+-------|--------------------
+[2:0]  | Read Status
+[3]    | Read JEDEC ID
+[4]    | Read SFDP
+[10:5] | Read commands
+
+When Read Status, Read JEDEC ID commands are processed in Flash mode, the other fields in the command information entry are not used.
```

Comment:
just to make sure i understand, if we are in flash mode, OR we are in passthrough mode with intercept_en set, the other fields of these configurations are ignored right? If what i'm saying is true, it may be clearer to describe it that way.

Reply:
Right. Thanks for pointing it out.